apps/freelance/src/redux/auth.api.ts

```diff
@@ -0,0 +1,44 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+interface IForm {
+  email: string;
+  password: string;
+}
+
+interface IServerResponse {
+  id: string;
+  email: string;
+  isActivated: boolean;
+}
+
+interface IToken {
+  token: string;
+}
+
+const baseUrl: string = process.env['NX_API_URL_AUTH'] || '';
```

Comment:
it should be in constant file. in other case this condition will be in each API file, it does not make any sence

Reply:
Updated